src/client/theme-default/styles/vars.css

```diff
@@ -6,4 +6,9 @@
   --text-color-light: #476582;
   --code-bg-color: #282c34;
   --accent-color: #3eaf7c;
+
+  /* responsive breakpoints */
+  /* --mq-narrow: 959px; */
+  /* --mq-mobile: 719px; */
+  /* --mq-mobile-narrow: 419px; */
 }
```

Comment:
I cannot use CSS variables in media query.
This is just memos.

We should manage to share break points variables using other solution.